hooks/checkout

```diff
@@ -27,4 +27,14 @@ if [ "${DRY_RUN}" != "true" ]; then
   # Clone the repository
   git clone "${REPOSITORY_URL_OPTION}" "${CHECKOUT_PATH_OPTION}" \
     --config core.sshCommand="ssh -i ${KEY_NAME} -o StrictHostKeyChecking=no -y"
+
+  cd "${CHECKOUT_PATH_OPTION}"
+
+  # Clean up the clone
+  git clean -ffxdq
+
+  # Checkout the branch
+  if [ "${BUILDKITE_BRANCH}" != "" ]; then
+    git fetch -v --prune -- origin "refs/heads/${BUILDKITE_BRANCH}"
```

Comment:
I think you also need to do something like `git checkout "origin/${BUILDKITE_BRANCH}"` or `git checkout "${BUILDKITE_COMMIT}"`

Reply:
Of course! I knew there was something I'd missed, have added this in now, thank you!